### ISSUE TITLE
Steam overlay doesn't work when launching through MO2

### ISSUE THREAD

**Issue by Wadu436** [TanninOne/modorganizer#439]
It doesn't work when launching Fallout4.exe directly or through F4SE. It does work when launched through Fallout 4 Launcher and through steam.

**Comment by shepherdlester** [comment]
I have this exact problem but with skyrim and iv gotten no help with it https://www.youtube.com/watch?v=GvnXkiHGqyw

**Comment by Wadu436** [comment]
No, my problem is that Steam Overlay doesn't open, and the FPS counter doesn't show, even though Steam recognizes I'm playing Fallout 4 (Alt+Tab to check the Steam Client)

**Comment by lithiumfox** [comment]
It seems to only happen to (some) people. Works fine for me, for example.

**Comment by shepherdlester** [comment]
My mistake.

**Comment by shepherdlester** [comment]
I'ts extremely frustrating the lack of support iv been getting. I've posted multiple threads to the nexus help boards with a very detailed description of my problem and have gotten almost no help whatsoever.

**Comment by Wadu436** [comment]
Tannin is probably a student, so he probably has exams around this time (just a guess)

**Comment by TanninOne** [comment]
I'm not, I've been working for years.
But in the precious little spare time I have left I want to advance the software not answer question. There are around 360 open issues here, posts on nexus, step, reddit, a german modding forum, I get mails, I get PMs, ... How do you expect me to reply to everything? And especially: How do you expect me reply to everything and get any work done?

If you get frustrated because of bugs in an alpha or think you are entitled to snappy support for a free software you really need a reality check. I've released this alpha so YOU can help ME find bugs quicker, not so I can hold your hand while you moronically try to use unfinished software "productively".

So since I'm one of the people who don't have this problem, how about you guys try to investigate what differentiates the affected systems from the unaffected? How about someone affected posts system specs and a log file?

**Comment by shepherdlester** [comment]
settle down little guy, i know you mod author pricks are overwhelmed. i've been looking for help from expert modders on the nexus forum help boards, not you. I posted here because this is where i was directed to on your page.

**Comment by shepherdlester** [comment]
"Skyrim  Mod Troubleshooting" and "Skyrim Technical Support" for instance. i think i've learned my lesson even though I've been polite and provided alot of information on my problem I havent gotten any useful information. but i dont blame them they either dont understand the problem or dont feel like bothring. y'know taking the time of day and thats fine nobody has to help its not urgent. you've been working on it for years and i shouldnt expect it to work okay. well i'll be using NMM from now in then.

**Comment by shepherdlester** [comment]
maybe i'm being reactionary but i dont wanna be basically called an asshole for looking for help with your product and then still be expected to grovel at the self importent modders feet for help

**Comment by TanninOne** [comment]
You and I have very different opinions on what "polite" is and how "a lot of infomation" look. If you don't intend to contribute then please at least stop posting, this is an issue tracker and not a discussion forum.

**Comment by shepherdlester** [comment]
you're right.

**Comment by WhiteZeroX** [comment]
Watching this from the outside, seems to be both @TanninOne and @shepherdlester are both being fairly rude. Tannin's "moronically" quip and shep's other various insults. C'mon people, keep it civil.

**Comment by grasmann** [comment]
@WhiteZeroX
It is quite frustrating though if people keep asking about the same thing and you don't have that issue.
You can't effectively fix a problem you can't reproduce.
And considering that it's a minor thing makes it all the more annoying.

**Comment by NoShotz** [comment]
@shepherdlester MO2 is an alpha it is expected to have bugs, he has only been working on MO2 for about a month.

**Comment by WhiteZeroX** [comment]
@grasmann No doubt it can be frustrating and I don't really blame Tannin for getting overwhelmed, doing this in his freetime. It's a popular project, so its going to attract a lot of people, and not everyone understands that development can be a bit slow going when basically one person is working on it bit by bit. But I also don't feel that lashing out against your users is a great way to go about it either.

**Comment by TanninOne** [comment]
Guys, sorry but I opted to delete several posts I felt were off topic to this issue.
Please let me say again: This is an issue tracker not a forum. The only on-topic posts are those that have a chance to help fix the issue. Other discussions have no place here.

If you can contribute to the issue being fixed then your contribution is welcome, no matter how it's phrased. If you're not planning to help: please move along, there is nothing to see here. And no, you don't have to leave a comment on your way out.

**Comment by Wadu436** [comment]
Alright, it seems the fault is with RivaTuner Statistics Server (an overlay program that starts automatically alongside MSI Afterburner). I don't know if this also happens with FRAPS or other overlays.

**Comment by TanninOne** [comment]
Awesome find wadu, I'll try to reproduce this tomorrow.

**Comment by WhiteZeroX** [comment]
@Wadu436 FRAPS works fine with MO + Steam Overlay here on Windows 8.1

**Comment by Wadu436** [comment]
Oh yeah should probably report OS too. I'm on Windows 10

**Comment by grasmann** [comment]
I'm using windows 10 as well.
Also I'm using MSI Afterburner and ENBoost and steam overlay is working for me.

**Comment by FenghuangZero** [comment]
I see this bug is still open, so that saves me from making a similar one, although my problem is slightly different as I am trying to run Oblivion. I don't have any of the other overlay programs mentioned above.

This is likely a known issue with Oblivion, but I am posting here to bring attention to the fact that the previous workaround is no longer available.

The steam overlay is required for Oblivion Script Extender to run, however the overlay does not function. Previously, it was possible to run the Steam.exe Executable with the "-applaunch 22330" command line argument, combined with the Script Extender Load Mechanism to work around this, however the alternate Load Mechanism is no longer working in 2.0 onwards. The hook.dll and mo_path.txt files that should be created in the OBSE/Plugins folder are not created, and attempting to use a version of hook.dll from 1.3.11 causes a CTD on launch.

**Comment by mano-1** [comment]
I know Tannin is not developing MO2 anymore, but there are several users who still using MO2 for FO4 or SEE64...

When launching SEE (_SkyrimSE.exe_) or FO4 (_Fallout4.exe_) through MO2, Steam overlay does not work. I can not take screenshots, see my achieviments etc.

BUT! When launching the **SkyrimSELauncher.exe** or **Fallout4Launcher.exe** through MO2 (there is a bethesda "splash" (Play, Options, Support, Exit) and the Steam overlay works without any problem!

I don't know, hope thats helps. :)

**Comment by ZeroGravity83** [comment]
I had the same issue. But running Steam as an Administrator fixed it.

**Comment by FenghuangZero** [comment]
Sadly I now think my issue in particular may be related to the Windows 10 preview updates. Overlays aren't working for a lot of games and I think it might have something to do with Microsoft's 'One OS over all devices' plan and adding SMS/telephone capability
